api/room/schema.py

```diff
@@ -147,7 +147,32 @@ def mutate(self, info, room_id, **kwargs):
         return DeleteRoom(room=exact_room)
 
 
+class IncrementTotalRoomFeedbackView(graphene.Mutation):
+
+    class Arguments:
+        room_id = graphene.Int(required=True)
+    room = graphene.Field(Room)
+
+    def mutate(self, info, **kwargs):
+        validate_empty_fields(**kwargs)
+        query = Room.get_query(info)
+        exact_room = query.filter_by(id=kwargs['room_id']).first()
+        if not exact_room:
+            raise GraphQLError("Non-existent room id")
+
+        if exact_room.total_room_feedback_views:
+            new_total_feedback_view = exact_room.total_room_feedback_views + 1
+        if not exact_room.total_room_feedback_views:
+            new_total_feedback_view = 1
```

Comment:
Is there a reason for this? Why not just see 0 as the total number of views?

Reply:
Initially, the DB has NULL set for this field and I looked up ways of changing this to a value of 0 such that I could do the incrementing from the start. I was not able to find one unfortunately hence the second if-statement.

Context: Addition cannot be done on `exact_room.total_room_feedback_views` when it is NULL.

Reply:
@bevkololi I think what this code is doing is to increment the number of views. So that is why he is adding a 1 to the number of views in the first if statement and setting it to 1 in the second if statement since the current number of views there is 0 or NULL.

Reply:
Got it @kampkelly, @RonKbS